browser/brave_content_browser_client.cc

```diff
@@ -597,35 +598,43 @@ BraveContentBrowserClient::CreateURLLoaderThrottles(
   auto result = ChromeContentBrowserClient::CreateURLLoaderThrottles(
       request, browser_context, wc_getter, navigation_ui_data,
       frame_tree_node_id);
-#if BUILDFLAG(ENABLE_SPEEDREADER)
-  using DistillState = speedreader::DistillState;
   content::WebContents* contents = wc_getter.Run();
-  if (!contents) {
```

Comment:
If the Speedreader buildflag is set, but we don't get a WebContents, then wouldn't we also skip the Debounce throttle? I'm guessing that would never happen which is why this works, and I moved this block of code around in my PR, but just want to double check this is expected @pilgrim-brave

Reply:
We should definitely get the WebContents outside of any conditional macro